Source/WTF/wtf/text/ASCIILiteral.h

```diff
@@ -145,17 +141,16 @@ constexpr ASCIILiteral operator"" _s(const char* characters, size_t)
 
 constexpr std::span<const LChar> operator"" _span(const char* characters, size_t n)
 {
+    auto span = std::span { bitwise_cast<const LChar*>(characters), n };
```

Comment:
I'm surprised the compiler didn't complain here about using the two argument span constructor. But maybe it's smart enough to know the provenance of the 'n' passed to operator""?

If the compiler does ultimately complain about this, we'll need to add the unsafeForgeSpan function I described in Slack.

Reply:
I just realized that I'm using an older clang compiler from macOS 14.x, not one from macOS 15, so the newer clang may find more issues.  (This is going to be an issue if EWS bots aren't updated to macOS 15 soon.)

I may need to abandon this PR since I was trying to do a small fix, but it's turning into a bigger fix.  :(

Reply:
Nevermind, I tested on macOS 15 and have a fix.